products/vmware_cb_response.py

```diff
@@ -8,17 +8,25 @@
 
 class CbResponse(Product):
     product: str = 'cbr'
+    profile: str = 'default'
```

Comment:
⛏️ Let's go ahead and also add the new variables here
```suggestion
    profile: str = 'default'
    url: Optional[str] = None
    token: Optional[str] = None
    _sensor_group: Optional[list[str]] = None
```